src/main/java/io/jenkins/plugins/kubernetes/NoDelayProvisionerStrategy.java

```diff
@@ -30,12 +30,12 @@
 public class NoDelayProvisionerStrategy extends NodeProvisioner.Strategy {
 
     private static final Logger LOGGER = Logger.getLogger(NoDelayProvisionerStrategy.class.getName());
-    private static final boolean DISABLE_NODELAY_PROVISING = Boolean.valueOf(
+    private static final boolean DISABLE_NO_DELAY_PROVISIONING = Boolean.parseBoolean(
             System.getProperty("io.jenkins.plugins.kubernetes.disableNoDelayProvisioning"));
 
     @Override
     public NodeProvisioner.StrategyDecision apply(NodeProvisioner.StrategyState strategyState) {
-        if (DISABLE_NODELAY_PROVISING) {
+        if (DISABLE_NO_DELAY_PROVISIONING) {
```

Comment:
Keep the diff smaller by avoiding gratuitous changes. Can always file separate stylistic cleanup PRs.